op.h

```diff
@@ -171,6 +171,13 @@ Deprecated.  Use C<GIMME_V> instead.
    op_convert_list to set op_folded.  */
 #define OPf_FOLDED      (1<<16)
 
+/* force newSTATEOP() to make an OP_NEXTSTATE.
```

Comment:
I would like to see: "... as opposed to an OP_DBSTATE"

Reply:
"make an OP_NEXTSTATE even when in debugging mode and a OP_DBSTATE would normally be produced"

Reply:
I reworded it a bit more than that.